cypress/integration/pages/frontPage/testsForCanonicalOnly.js

```diff
@@ -13,6 +18,20 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
     it('should not have an AMP attribute', () => {
       cy.get('html').should('not.have.attr', 'amp');
     });
+    // Once the most read is implemented for amp and all other pages this test will be moved to testforallpages
+    if (serviceHasMostRead(service) && Cypress.env('APP_ENV') === 'local') {
+      it('should contain most read component if the toggle is enabled', () => {
+        if (useAppToggles.mostRead) {
+          cy.get('[aria-labelledby="Most-Read"]')
+            .should('be.visible')
+            .within(() => {
+              cy.get('li')
+                .should('have.length.of.at.least', 1)
```

Comment:
Can we check the number of items against what is in the config. i.e: https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/services/arabic.js#L167